src/data/nat/basic.lean

```diff
@@ -275,6 +275,9 @@ theorem of_le_succ {n m : ℕ} (H : n ≤ m.succ) : n ≤ m ∨ n = m.succ :=
 lemma succ_lt_succ_iff {m n : ℕ} : succ m < succ n ↔ m < n :=
 ⟨lt_of_succ_lt_succ, succ_lt_succ⟩
 
+lemma succ_succ_ne_one (n : ℕ) : n.succ.succ ≠ 1 :=
+  by { rintros h, rw one_succ_zero at h, simp only at h, apply succ_ne_zero n h, }
```

Comment:
```suggestion
succ_ne_zero n ∘ succ.inj
```
Should this really be in the library?

Reply:
I think zulip said that `norm_num` was best for this type of thing.

Reply:
I have removed it.